.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!-- Thank you for the contribution! -->
+
+### Why do we need this PR
```

Comment:
maybe we can skip this field why do we need this PR as the issue should describe why we need the PR?